.orchestra/support/js-prerequisites/package-lock.json

```diff
@@ -0,0 +1,77 @@
+{
```

Comment:
I thought we'd be generating this. :uhm:

Reply:
I've double checked, using just the `package.json` with **exact** versions will give deterministic results.

Reply:
Ah, right we're not in revng here. This is to populate our "registry".